app/ceom/maps/templates/maps/pixel_validation.html

```diff
@@ -110,6 +118,95 @@ <h2 class="legend title">Pixels Validation</h2>
             fillOpacity: 0,
             clickable: false,                 
         });
+        
+        let centers = JSON.parse('{{ centers_list|safe }}');
+        polygonData = [];
+        centers.forEach(center => {
+            let coords = [center[0], center[1]];
+            let grid_npix = center[2];
+
+            let sin2latlon = function(ih, iv, xi, yi, npix) {
```

Comment:
Putting the function definitions inside a forEach loop here is a little unusual, but I think JavaScript is so loosy-goosy it shouldn't matter too much